libs/wingsdk/src/cloud/function.ts

```diff
@@ -96,20 +96,26 @@ export abstract class FunctionBase extends Resource implements IInflightHost {
     const tempdir = mkdtemp();
     const outfile = join(tempdir, "index.js");
 
-    esbuild.buildSync({
-      bundle: true,
-      stdin: {
-        contents: lines.join("\n"),
-        resolveDir: tempdir,
-        sourcefile: "inflight.js",
-      },
-      target: "node16",
-      platform: "node",
-      absWorkingDir: tempdir,
-      outfile,
-      minify: false,
-      external: ["aws-sdk"],
-    });
+    try {
+      esbuild.buildSync({
+        bundle: true,
+        stdin: {
+          contents: lines.join("\n"),
+          resolveDir: tempdir,
+          sourcefile: "inflight.js",
+        },
+        nodePaths: [join(__dirname, "..", "..", "node_modules")],
+        target: "node16",
+        platform: "node",
+        absWorkingDir: tempdir,
+        outfile,
+        minify: false,
+        logLevel: "silent",
+        external: ["aws-sdk"],
+      });
+    } catch (e) {
```

Comment:
I this `catch` all we really needed to avoid the terminal garbling?

Reply:
It's the loglevel: "silent" above

Reply:
Looks like the issue is still present for me @MarkMcCulloh